cfgov/unprocessed/js/molecules/Expandable.js

```diff
@@ -48,9 +48,12 @@ function Expandable( element ) { // eslint-disable-line max-statements, inline-c
   /**
    * @param {number} state
    *   Allows passing of EXPANDED flag to set expanded state.
-   * @returns {Object} The Expandable instance.
+   * @returns {Expandable|undefined} An instance,
+   *   or undefined if it was already initialized.
    */
   function init( state ) {
+    if ( !atomicHelpers.setInitFlag( _dom ) ) { var inst; return inst; }
```

Comment:
Okay, mostly did this because that variable is limited to the if block and not elsewhere and it wouldn't require importing a module to copy this line to new components.